recipes/r-beyondcell/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+r_base:
+  - 4.0.0
```

Comment:
Is this incompatible with 4.1 or is this instead an issue with getting the environment to solve?

Reply:
It is compatible with R 4.1.0 but in the meta.yaml we must be modified to include r-reshape2>=1.4.4